bloom/config.py

```diff
@@ -204,7 +204,9 @@ def __str__(self):
         'git-bloom-generate -y rosdebian --prefix release/:{ros_distro}'
         ' :{ros_distro} -i :{release_inc} --os-name debian --os-not-required',
         'git-bloom-generate -y rosrpm --prefix release/:{ros_distro}'
-        ' :{ros_distro} -i :{release_inc}'
+        ' :{ros_distro} -i :{release_inc} --os-name fedora',
+        'git-bloom-generate -y rosrpm --prefix release/:{ros_distro}'
+        ' :{ros_distro} -i :{release_inc} --os-name rhel',
```

Comment:
@cottsay have you considered whether we should use the `--os-not-required` option in the default actions for fedora or rhel as we do for debian above?

Reply:
That option is only for the Debian generator: https://github.com/ros-infrastructure/bloom/search?q=os_not_required

The RPM generator always acts as if `--os-not-required` was specified.